src/styles/header.scss

```diff
@@ -0,0 +1,74 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.head {
+  bottom: 50px;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  align-items: center;
+  justify-content: space-between;
+  background: #fff;
+  width: 100%;
+}
+
+.logo__img {
+  display: block;
+  width: 40px;
+  height: 40px;
+  align-self: center;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  color: #000;
+  position: relative;
+  display: block;
+  padding: 23px 0;
+  font-weight: 500;
+  line-height: 14px;
+  font-size: 12px;
+  text-transform: uppercase;
+  text-decoration: none;
+}
+
+.nav__link[data-qa="nav-hover"] {
+  color: #00acdc;
```

Comment:
Consider using variables for repeating colors

Reply:
If I use a variable for this color here, my hover breaks

Reply:
its impossible, variable cant break up you hover